settings.gradle

```diff
@@ -2,7 +2,7 @@
  * Gradle's setting file which references all modules.
  *
  * @author Armin Schnabel
- * @version 1.0.1
+ * @version 1.1.0
  * @since 1.0.0
  */
-include ':app', ':datacapturing', ':persistence', ':synchronization', ':testutils'
```

Comment:
This "dummy" app module is generated when a new project is created in Android Studio. I didn't dare to delete it all those years ;-) But now I'm removing it finally as I don't see any reasons why we need a dummy "app" module in the library projects.

Reply:
Was the same for me. I thought hell would break lose if that is deleted. ;)